src/lib.rs

```diff
@@ -115,10 +117,24 @@ pub struct PetSpirv<'spir> {
 
 pub fn export_spirv_cfg(module: &SpirvModule) {
     let mut file = File::create("test.dot").expect("file");
+    writeln!(file, "digraph {{");
     for f in &module.module.functions {
+        let mut has_control_flow = false;
+        for block in f.basic_blocks.iter() {
+            let last_opcode = block.instructions.last().unwrap().class.opcode;
+            if last_opcode == spirv::Op::BranchConditional || last_opcode == spirv::Op::Switch {
+                has_control_flow = true;
```

Comment:
This could `break` - or could even be written like this:
```rust
let has_control_flow = f.basic_blocks.iter().any(|block| {
    matches!(
        block.instructions.last().unwrap().class.opcode,
        spirv::Op::BranchConditional | spirv::Op::Switch
    )
});
```